doc/api/util.md

```diff
@@ -535,6 +542,34 @@ console.log(inspect(weakSet, { showHidden: true }));
 // WeakSet { { a: 1 }, { b: 2 } }
 ```
 
+The `sorted` option makes sure the output is stable, no matter of the properties
```

Comment:
It seems the word `stable` is a bit confusing here. Compare this sentence with the [wiki definition](https://en.wikipedia.org/wiki/Sorting_algorithm#Stability) (referenced from the MDN `Array.prototype.sort()` page):

> Stable sort algorithms sort identical elements in the same order that they appear in the input.

I.e. the traditional meaning of the `stable` sorting is opposite to `no matter of the properties insertion order`.

Reply:
True, I think I found a better wording.